firmware/thruna/BMS/Src/App/App_Accumulator.c

```diff
@@ -9,6 +9,12 @@
 
 #define NUM_AUX_THERMISTORS (6U)
 
+// What we will compare against to determine if a segment has a blown fuse and is
+// less than three cells in parallel. We calculate number of cells and compare.
+// TODO: TUNING AND TESTING REQUIRED, STILL A WORK IN PROGRESS THRESHOLD.
+#define BLOWN_FUSE_THRESHOLD (2.5f)
+#define FUSE_CHECK_CURRENT_THRESHOLD (40.0f)
```

Comment:
Always good to specify units where applicable:
```suggestion
#define FUSE_CHECK_CURRENT_THRESHOLD_AMPS (40.0f)
```